pkgs/top-level/all-packages.nix

```diff
@@ -18310,6 +18310,8 @@ with pkgs;
 
   openai = with python3Packages; toPythonApplication openai;
 
+  openaiauth = with python3.pkgs; toPythonApplication openaiauth;
```

Comment:
Please remove this, the package does not include any runnable, it is a pure library package.

Reply:
done